assembly/assemblyCommon/AssemblyUEyeWidget.cc

```diff
@@ -23,8 +23,8 @@ AssemblyUEyeWidget::AssemblyUEyeWidget(AssemblyVUEyeModel *model,
       model_(model)
 {
     // Connect all the signals
-    connect(model_, SIGNAL(cameraCountChanged(unsigned int)),
-            this, SLOT(cameraCountChanged(unsigned int)));
+    connect(model_, SIGNAL(cameraCountChanged(2)),
+            this, SLOT(cameraCountChanged(2)));
```

Comment:
why is the camera count hardcoded when signaled to the widget?

Reply:
Good point. That shouldn't be hardcoded. The edit was made long ago during the initial setup and the original author left our group. We're doing some digging to understand why that edit was done in the first place, if it's still needed for our setup and if that's the case find a non-hardcoded solution.

Reply:
Agreed this shouldn't be hard coded. I'll investigate further if this break's out set up tomorrow in the lab as well as search through old notes to see if I can find the reason for the edit